HeaderFooter.php

```diff
@@ -7,7 +12,7 @@
  */
 
 # Credits
-$GLOBALS['wgExtensionCredits']['other'][] = array( 
+$wgExtensionCredits['other'][] = array( 
```

Comment:
I'm guessing Composer uses `extension.json` if it's available, so having things in `$GLOBALS['...']` format is no longer required?